.eslintrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "root": true,
+  "extends": ["@open-wc/eslint-config", "prettier"],
+  "rules": {
+    "import/extensions": "off",
```

Comment:
Waarom enforecen we hier niet dat hij alle extensies er af stript?

Reply:
Dat was een rule dat die verlicht `accordion/index.js` verwachtte, ipv `accordion`

Reply:
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/extensions.md